src/services/apiEmployees.js

```diff
@@ -0,0 +1,7 @@
+import axios from "axios";
+export async function getEployees() {
```

Comment:
Need to use URL from environment variable so that it will be generated at build time depending on dev or prod

Reply:
Corrected